packages/perseus/src/widgets/numeric-input/numeric-input.testdata.ts

```diff
@@ -36,6 +37,305 @@ export const question1: PerseusRenderer = {
     },
 };
 
+// Used for Storybook.
+export const defaultQuestion: PerseusRenderer = {
+    content: "$5008 \\div 4 =$ [[\u2603 numeric-input 1]] ",
```

Comment:
It would be super easy to turn these into generators which would reduce the risk of accidentally mutating test data:

```suggestion
export function getDefaultQuestion(): PerseusRenderer {
    return {
        content: "$5008 \\div 4 =$ [[\u2603 numeric-input 1]] ",
```